Tests/.editorconfig

```diff
@@ -150,3 +150,9 @@ resharper_expression_is_always_null_highlighting = none
 
 # ReSharper properties
 resharper_keep_user_linebreaks = true
+
+# Use element access. Suggestion because it's not always faster
+dotnet_diagnostic.RCS1159.severity = none
```

Comment:
[RCS1159](https://github.com/JosefPihrt/Roslynator/blob/main/docs/analyzers/RCS1159.md) is about using `EventHandler<T>`?